ddpui/tests/api_tests/test_public_report_api.py

```
@@ -0,0 +1,467 @@
+        assert public_snapshot.public_access_count == initial_count + 2
+
+
+# ================================================================================
+# Test get_public_report_chart_data
+# ================================================================================
+
+
+class TestGetPublicReportChartData:
+    """Tests for get_public_report_chart_data endpoint"""
+
+    def test_invalid_token(self, seed_db):
```

Ruff (0.15.5)

[error] Redefinition of unused `seed_db` from line 39: `seed_db` redefined here

(F811)